docs/learn/encyclopedia/lumen-supply-metrics.mdx

```diff
@@ -39,7 +41,7 @@ As of December 12, 2019, the Dashboard API shows:
 
 **inflationLumens** For the first five or so years of Stellar’s existence, the supply of lumens increased by 1% annually. This “network inflation” was ended by validator vote on October 28, 2019. The total number of lumens generated by inflation was 5,443,902,087.3472865.
 
-Adding this number to the Original Supply, you get the total lumens that have ever existed: 105,443,902,087.3472865. This number is visible on the List All Ledgers (`https://horizon-mainnet-url/ledgers?order=desc`) Horizon API endpoint as \_embedded.records.total_coins.
+Adding this number to the Original Supply, you get the total lumens that have ever existed: 105,443,902,087.3472865. This number is visible on the List All Ledgers (`https://horizon.stellar.org/ledgers?order=desc`) Horizon API endpoint as \_embedded.records.total_coins.
```

Comment:
We could link out to pages in our docs and to the Horizon community providers:

The number is visible on the [List All Ledgers](https://developers.stellar.org/network/horizon/api-reference/resources/list-all-ledgers) Horizon API as \_embedded.records.total_coins when called from a mainnet Horizon(https://developers.stellar.org/network/horizon/horizon-providers).